playbook.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- hosts: all
+  become: yes
+  roles:
+    - common
+    - zooplus
+  vars:
+    user: arcones
```

Comment:
To make it more easy to use you can use extra args

Reply:
done for the username